gtfs/CHANGES.md

```diff
@@ -4,34 +4,37 @@ The GTFS Specification is not set in stone. Instead, it is an open specification
 The official specification, reference and documentation are written in English. If a translation to a different language differs from the English original, the latter takes precedence. All communication is performed in English.
 
 1. Create a git branch with update of all relevant parts of protocol definition, specification and documentation files (except for translations).
-1. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
```

Comment:
Something weird happened here with the formatting - it's showing the entire block being deleted/added.  It should show just individual line changes, as in the gtfs-realtime/CHANGES.md file above.

Reply:
The markdown formatting is currently incorrect so automatic numbering and indented bullets are not properly rendered. In the commit I added a few spaces, tabs and line breaks to get things to render correctly. Perhaps that's what's causing the issue?

Reply:
@barbeau just to clarify, we don't need to call a vote on this formatting-only PR, correct?

Reply:
Yes, I think as long as we make it clear that it's a formatting change only, one of us can approve/merge without a vote.